edx_proctoring/templates/proctoring/seq_proctored_practice_exam_submitted.html

```diff
@@ -0,0 +1,52 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You have submitted this practice proctored exam
+    {% endblocktrans %}
+  </h3>
+
```

Comment:
@chrisndodge Same as for seq_proctored_exam_submitted.html:
Add a text paragraph here, under the h3 and before the h4?

<p>Make sure you return to the proctoring software and select **Quit** to end the proctoring session.</p>

..and I am also going to modify the text below the h4.